pkg/kubernetes/scripts/virtual-machines/utils.jsx

```diff
@@ -26,3 +26,12 @@ export function getPairs(object) {
         value: object[key]
     }))
 }
+
+export function mouseClick(fun) {
```

Comment:
Quite repeating bit in the last days, looking forward to have it already in somewhere :-)